### ISSUE TITLE
Username password length limit on client side should either be removed or the length limit should be decreased

### ISSUE THREAD

**Issue by tarun0** [openMF/android-client#688]
**Summary:** 

During Login, if the username and/or the password entered has a length less than 6 (5 for username), a message (snackbar) "Invalid password/username length" is shown.

It seems that there is no such requirement at the backend side. So, it should either conform to the restrictions imposed by backend or should be removed.

**Steps to reproduce:** 
Open Login Activity. Enter any password/username having less than 6 characters.

**Comment by mayank-kgp** [comment]
@tarun0 Are you working on it?

**Comment by tarun0** [comment]
@mayank-kgp No, I haven't yet. I wanted to discuss the same with the members.

**Comment by therajanmaurya** [comment]
I will discuss with Markus

**Comment by AkshGautam** [comment]
@tarun0 @therajanmaurya what's the outcome of this discussion? do we need to remove such restrictions on the client side? I would like to work on this if the community has reached a conclusion that backend does not support the restrictions
